src/fenicsxconcrete/experimental_setup/am_multiple_layers.py

```diff
@@ -0,0 +1,152 @@
+import logging
+from collections.abc import Callable
+
+import dolfinx as df
+import numpy as np
+import pint
+import ufl
+from mpi4py import MPI
+from petsc4py.PETSc import ScalarType
+
+from fenicsxconcrete.boundary_conditions.bcs import BoundaryConditions
+from fenicsxconcrete.boundary_conditions.boundary import plane_at
+from fenicsxconcrete.experimental_setup.base_experiment import Experiment
+from fenicsxconcrete.helper import Parameters
+from fenicsxconcrete.unit_registry import ureg
+
+
+class AmMultipleLayers(Experiment):
+    """sets up a simple layered structure
+
+    all layers of the same height are on top of each other, the boundary on the bottom is fixed
+    the mesh includes all (activation via pseudo-density)
+
+    """
+
+    def __init__(self, parameters: dict[str, pint.Quantity]):
+        """defines default parameters, for the rest, see base class"""
+
+        # initialize default parameters for the setup
+        default_p = Parameters()
+        # default_p['dummy'] = 'example' * ureg('')  # example default parameter for this class
+
+        # updating parameters, overriding defaults
+        default_p.update(parameters)
+        self.logger = logging.getLogger(__name__)
+        super().__init__(default_p)
+
+    @staticmethod
+    def default_parameters() -> dict[str, pint.Quantity]:
+        '''set up a set of working values as example"""
+
+        Returns: dictionary with required parameter
+
+        '''
+        # this must de defined in each setup class
+
+        setup_parameters = {}
+        # geometry
+        setup_parameters["dim"] = 2 * ureg("")
+        setup_parameters["num_layers"] = 10 * ureg("")  # number of layers in y
+        setup_parameters["layer_length"] = 0.5 * ureg("m")  # x_dimension
+        setup_parameters["layer_height"] = 0.01 * ureg("m")  # Dy dimension
+        # only relevant for 3D case [z-dimension]
+        setup_parameters["layer_width"] = 0.05 * ureg("m")
+
+        # mesh
+        setup_parameters["num_elements_layer_length"] = 10 * ureg("")
+        setup_parameters["num_elements_layer_height"] = 1 * ureg("")
+        # only relevant for 3D case
+        setup_parameters["num_elements_layer_width"] = 2 * ureg("")
+
+        # only relevant for 2D case
+        # p["stress_case"] = "plane_strain"
```

Comment:
why is this commented?

Reply:
because until now I'm not sure how the new parameter class deals with str arguments (since pint is required!) or where I have to define such information now. For the moment where I just test the set-up with the linear elastic material, I don't need the parameter.